CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+# [2.0.0](https://github.com/patternfly/patternfly-next/compare/v1.0.250...v2.0.0) (2019-04-08)
+
+
+### Bug Fixes
+
+* **a11y:** remove aria-required, remove required attr from labels ([#1642](https://github.com/patternfly/patternfly-next/issues/1642)) ([d5b3c40](https://github.com/patternfly/patternfly-next/commit/d5b3c40))
+* **card:** refactor child flex/spacing selector ([#1675](https://github.com/patternfly/patternfly-next/issues/1675)) ([6fbf3a5](https://github.com/patternfly/patternfly-next/commit/6fbf3a5))
+* **datalist:** change grid modifier on nested compact table ([#1698](https://github.com/patternfly/patternfly-next/issues/1698)) ([ba34f27](https://github.com/patternfly/patternfly-next/commit/ba34f27))
+* **DataListDemo:** use page-main-nav component ([#1648](https://github.com/patternfly/patternfly-next/issues/1648)) ([89f8a56](https://github.com/patternfly/patternfly-next/commit/89f8a56))
+* **table:** remove unnecessary aria-label ([#1643](https://github.com/patternfly/patternfly-next/issues/1643)) ([11df699](https://github.com/patternfly/patternfly-next/commit/11df699))
+* **wizard:** update disabled step, remove right align option ([#1661](https://github.com/patternfly/patternfly-next/issues/1661)) ([fc6175a](https://github.com/patternfly/patternfly-next/commit/fc6175a))
+
+
+### Features
+
+* **form-control:** add ellipsis when text overflows ([#1619](https://github.com/patternfly/patternfly-next/issues/1619)) ([21e734d](https://github.com/patternfly/patternfly-next/commit/21e734d))
+* **mixin:** add mixin for hiding scrollbars ([#1676](https://github.com/patternfly/patternfly-next/issues/1676)) ([d225311](https://github.com/patternfly/patternfly-next/commit/d225311))
+* **nav:** add overflow styling to horizontal nav examples ([#1615](https://github.com/patternfly/patternfly-next/issues/1615)) ([bfa521c](https://github.com/patternfly/patternfly-next/commit/bfa521c))
+* **page:** add fill/no-fill modifiers for main-section elements ([#1689](https://github.com/patternfly/patternfly-next/issues/1689)) ([e300ccf](https://github.com/patternfly/patternfly-next/commit/e300ccf))
+* **tabs:** add overflow styles for secondary tabs ([#1606](https://github.com/patternfly/patternfly-next/issues/1606)) ([5e34dfe](https://github.com/patternfly/patternfly-next/commit/5e34dfe))
+* **wizard:** add support for finished state ([#1672](https://github.com/patternfly/patternfly-next/issues/1672)) ([14cc898](https://github.com/patternfly/patternfly-next/commit/14cc898))
+
```

Comment:
Should we add this? It was tagged as `refactor()` but should be `feat()`

```suggestion
* **card:** allow for multiple body, add grow modifier ([#1484](https://github.com/patternfly/patternfly-next/pull/1484)) ([aa04c8a](https://github.com/patternfly/patternfly-next/commit/aa04c8ab9d0015b62479cafa38b8a626e9f475d0))
```

Reply:
It's down in the bottom with the breaking changes

Reply:
That entry doesn't describe the changes I mentioned though. Only the padding modifications from that PR are noted under the breaking changes section, since those are breaking changes. We now support `.pf-m-no-fill` on a card body to keep it from growing (card bodies grow by default), and added to the documentation that you can have multiple card body elements in a card. Those aren't breaking changes.